src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,64 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { UserInfo } from '../UserInfo';
+import classNames from 'classnames';
+import { Props } from './Props';
+export const TodoList: React.FC<Props> = React.memo(function TodoList({
+  todos,
+  selectedTodoId,
+  onDelete = () => {},
+  onSelect = () => {},
+}) {
+  return (
+    <React.Fragment>
+      <section className="TodoList">
+        <table className="table is-striped is-narrow box">
+          <thead>
+            <tr className="has-background-link-light">
+              <th>#</th>
+              <th>Title</th>
+              <th>User</th>
+              <th></th>
+              <th></th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {todos.map(todo => (
+              <tr
+                key={todo.id}
+                data-id={todo.id}
+                className={classNames({
+                  'has-background-info': selectedTodoId === todo.id,
+                })}
+              >
+                <td>{todo.id}</td>
+
+                <TodoInfo todo={todo} />
+
+                {false && <td>{todo.user && <UserInfo user={todo.user} />}</td>}
```

Comment:
The condition `false && <td>{todo.user && <UserInfo user={todo.user} />}</td>` will always evaluate to `false`, meaning the `UserInfo` component will never be rendered. If this is not intentional, consider revising the condition to achieve the desired behavior.